astroplan/core.py

```diff
@@ -110,7 +110,34 @@ class Observer(object):
     A container class for information about an observer's location and
     environment.
 
-    TODO: write this docstring
+    Examples
```

Comment:
Good! I would switch the order though for those who glance at this quickly -- we want them to first try to find their site in the site list and if it's not there show them how to create a site from scratch. That cool with you?